Neos.Flow/Tests/Unit/Package/PackageOrderResolverTest.php

```diff
@@ -319,24 +319,24 @@ public function packagesAndDependenciesOrder()
                     'symfony/console',
                     'doctrine/orm',
                     'imagine/imagine',
-                    'typo3/eel',
-                    'typo3/party',
-                    'typo3/fluid',
-                    'typo3/flow',
-                    'typo3/form',
-                    'typo3/typoscript',
-                    'typo3/neos-nodetypes',
-                    'typo3/imagine',
-                    'typo3/media',
-                    'typo3/extjs',
-                    'typo3/twitter-bootstrap',
-                    'typo3/setup',
-                    'typo3/typo3cr',
-                    'typo3/neos',
-                    'typo3/neosdemotypo3org',
+                    'neos/eel',
+                    'neos/party',
+                    'neos/fluid',
+                    'neos/flow',
+                    'neos/form',
+                    'neos/fusion',
+                    'neos/neos-nodetypes',
+                    'neos/imagine',
+                    'neos/media',
+                    'neos/extjs',
+                    'neos/twitter-bootstrap',
+                    'neos/setup',
+                    'neos/contentrepository',
+                    'neos/neos',
+                    'neos/demo',
                     'flowpack/behat',
-                    'typo3/kickstart',
-                    'typo3/sitekickstarter',
+                    'neos/kickstart',
```

Comment:
neos/kickstart**er** -> https://packagist.org/packages/neos/kickstarter

same in line: 149, 294 and 295